### PR TITLE
New package: YARC.YARCLauncher version v1.0.1

### LINKED ISSUE #189966: [Package Request]: YARC Launcher
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url:
Publisher: YARC
Package Name: YARC Launcher
Description: The official repository for the YARC launcher (a.k.a., YAL or Yet Another Launcher).
Package Version: v1.0.1
Installer Url: https://github.com/YARC-Official/YARC-Launcher/releases/download/v1.0.1/YARC.Launcher_1.0.1_x64_en-US.msi

### PR DESCRIPTION
**Pull request by fjwxzde**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.6.0 :rocket:

- close #189966
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189996)

### CODE REVIEW

File: manifests/y/YARC/YARCLauncher/v1.0.1/YARC.YARCLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  DisplayVersion: 1.0.1
+  ProductCode: '{CB26D07A-9EC2-4468-BB67-04754600765B}'
+  UpgradeCode: '{EE5DD577-83FE-594D-A069-A7955B8FD524}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\YARC Launcher'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/YARC-Official/YARC-Launcher/releases/download/v1.0.1/YARC.Launcher_1.0.1_x64_en-US.msi
+  InstallerSha256: 24A6FC2B93708F58BE0A452E10BE610B83E981A063FEB24D24CBDBB5AB530586
+  InstallerSwitches: {}
```
Comment (DuckDuckStudio) [inline manifests/y/YARC/YARCLauncher/v1.0.1/YARC.YARCLauncher.installer.yaml:31]:
```suggestion
```

**Review by DuckDuckStudio — CHANGES_REQUESTED** [review]

**Review by DuckDuckStudio — APPROVED** [review]

**Comment by fjwxzde** [comment]
Reopen this because [validation are not successful](https://dev.azure.com/shine-oss/winget-pkgs/_build/results?buildId=23870&view=logs&j=dc5484e7-078c-5e69-1cbb-0f132d85a80e&t=164321f8-a021-5d7c-6a6e-c77bb2b8a4bc).

**Comment by fjwxzde** [comment]
Hi @stephengillie ,
It looks like something's wrong, but I don't know why...

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 897.)

**Pull request closed without merging**